plugin/trino-druid/src/test/java/io/trino/plugin/druid/DruidSqlDataTypeTest.java

```diff
@@ -0,0 +1,180 @@
+/*
```

Comment:
`DruidSqlDataTypeTest` holds different `getPredicate`, I am thinking maybe we can pass the function via `create` instead of having a new class.

Reply:
I would prefer separate class. Let's not make things complicated for other connectors just for the sake of Druid.